src/common/components/signUpModal/signUpModal.tpl.html

```diff
@@ -1,53 +1,126 @@
 <div class="col-md-12 col-xs-12">
-  <form>
+  <div class="alert alert-danger" role="alert" ng-messages="$ctrl.signUpErrors" ng-if="$ctrl.hasError">
+    <div ng-message="400" translate>An error occurred during sign up.<br />Please Check your Email Address and Password and try again.</div>
+    <div ng-message="403" translate>An error occurred during sign up.<br />This is most likely caused by the use of a reserved domain name in your email address. You might try to
```

Comment:
403 FORBIDDEN - Email domain is reserved (cru.org, ccci.org, uscm.org...). There is an on boarding process for corporate email domains which can't be bypassed through give site.

These error messages are included in https://jira.cru.org/browse/EP-726 for Jason Chandler to modify.